### DISCUSSION TITLE
738FF7DE-52B2-22D3-D925-908178D6B5F9

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#142] [General]
# Sextans A

Sextans A, also known as DDO 75, is a small elliptical galaxy measuring about 5,000 light-years across. It is located at the Antila-Sextans subgroup about 4.4 million light-years away in the constellation Sextans

https://celestia.mobi/addon?item=738FF7DE-52B2-22D3-D925-908178D6B5F9&lang=es

**Comment by Pascualito1008** [comment]
Idk

**Comment by kmjordan122313-lgtm** [comment]
^_^
